addons/web/static/src/js/view_list.js

```diff
@@ -1872,7 +1872,7 @@ var Record = instance.web.Class.extend(/** @lends Record# */{
             if (typeof val !== 'object') {
                 output[k] = val;
             } else if (val instanceof Array) {
-                output[k] = val[0];
+                output[k] = val.length > 0 ? val[0] : null;
```

Comment:
Wouldn't `false` be slightly more consistent with backend? IMO JS `null` is akin to Python `None`, but an empty x2many in ORM is a `False`